src/components/tabs.md

```diff
@@ -172,11 +217,21 @@ Reference implementations are intended to demonstrate **what needs to be achieve
 
 ## Related research
 
-This topic does not yet have any related research available.
+During the development of the tabs implementation, the BBC accessibility team undertook research and hosted two days of usability testing with a number of blind and partially sighted participants.
```

Comment:
@Heydon BBC News undertook the research... with a number of keyboard only participants (this included participants with no vision impairment as well as a range of vision impairments)

Reply:
@greenc05 Thanks for checking. I'll incorporate the changes shortly.